includes/ft_printf_list.sh

```diff
@@ -15,17 +15,17 @@ then
 	CHK_FT_PRINTF_LIST+=("0" "1234")
 
 	# wrong flag
-	CHK_FT_PRINTF_LIST+=("0" "%")
-	CHK_FT_PRINTF_LIST+=("0" "% ")
-	CHK_FT_PRINTF_LIST+=("0" "% h")
-	CHK_FT_PRINTF_LIST+=("0" "%Z")
-	CHK_FT_PRINTF_LIST+=("0" "% hZ")
+	CHK_FT_PRINTF_LIST2+=("0" "%")
+	CHK_FT_PRINTF_LIST2+=("0" "% ")
+	CHK_FT_PRINTF_LIST2+=("0" "% h")
+	CHK_FT_PRINTF_LIST2+=("0" "%Z")
+	CHK_FT_PRINTF_LIST2+=("0" "% hZ")
 
 	# %
 	CHK_FT_PRINTF_LIST+=("0" "%%")
 	CHK_FT_PRINTF_LIST+=("0" "%5%")
 	CHK_FT_PRINTF_LIST+=("0" "%-5%")
-	CHK_FT_PRINTF_LIST+=("0" "%05%")
+	CHK_FT_PRINTF_LIST2+=("0" "%05%")
```

Comment:
Why should it be undefined behavior?
It does mean "at least 5 characters to print, right-aligned, filled with '0' on right-side", isn't it?